adbc.h

```diff
@@ -667,8 +786,80 @@ struct ADBC_EXPORT AdbcDriver {
                                          struct AdbcError*);
   AdbcStatusCode (*StatementSetSubstraitPlan)(struct AdbcStatement*, const uint8_t*,
                                               size_t, struct AdbcError*);
+
+  /// \defgroup adbc-1.1.0 ADBC API Revision 1.1.0
+  ///
+  /// Functions added in ADBC 1.1.0.  For backwards compatibility,
+  /// these members must not be accessed unless the version passed to
+  /// the AdbcDriverInitFunc is greater than or equal to
+  /// ADBC_VERSION_1_1_0.
+  ///
+  /// For a 1.0.0 driver being loaded by a 1.1.0 driver manager: the
+  /// 1.1.0 manager will allocate the new, expanded AdbcDriver struct
```

Comment:
Does the driver manager allocate anything? In `AdbcLoadDriver` it's the caller that passes the `AdbcDriver` pointer.

Reply:
If you use the `AdbcDatabaseInit` path (where the driver manager pretends to just be an ADBC driver, and you pass the driver name as one of the init options) then it will allocate for you, yes.